src/index.html

```diff
@@ -11,6 +11,42 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <label>
+      <form
+        class="inputForm top"
+        action="saerch"
+        method="get"
+        data-qa="big">
```

Comment:
read the 5th point from the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)